code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1778,6 +1778,8 @@
 	if(stat)
 		return
 	var/datum/category_group/underwear/UWC = input(usr, "Choose underwear:", "Show/hide underwear") as null|anything in global_underwear.categories
+	if(!UWC)
```

Comment:
If null is allowed as choice, we need to check for that before we try to access null.name